qnm/angular.py

```diff
@@ -105,7 +128,23 @@ def M_matrix_elem(s, c, m, l, lprime):
 
 # I don't know if this is necessary ... can just iterate
 def give_M_matrix_elem_ufunc(s, c, m):
-    """ TODO Document """
+    """ 
+    Parameters
+    ----------
+    s: integer
+      Spin-weight of interest
+
+    c: integer
+      Oblateness of the spheroidal harmonic
+
+    m: integer
+      Magnetic quantum number
+
+    Returns
+    -------
+    universal function (ufunc) object
+      Operates on arrays element by element
```

Comment:
Just `ufunc` would work for the type. The description should explain the meaning of the ufunc (i.e. ufunc that implements matrix elements of the spherical-spheroidal decomposition matrix) rather than "what is a ufunc"